docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/OneVersusAll.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.ML.SamplesUtils;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.MulticlassClassification
+{
+    public static class OneVersusAll
+    {
+        public static void Example()
```

Comment:
>Example [](start = 27, length = 7)

you probably want to link this file to extension method.

```
     /// <format type="text/markdown">
        /// <![CDATA[
        ///  [!code-csharp[SDCA](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/StochasticDualCoordinateAscentWithOptions.cs)]
        /// ]]></format>
        /// </example>
```

Reply:
no, this is about OVA not SDCA

---
In reply to: [271090986](https://github.com/dotnet/machinelearning/pull/3159#discussion_r271090986) [](ancestors = 271090986)